### ISSUE TITLE
`declare module '*.foo'` reported as unlisted dependencies

### ISSUE THREAD

**Issue by EvgenyOrekhov** [webpro/knip#511]
Steps to reproduce:

src/react-app-env.d.ts

```ts
/// <reference types="react-scripts" />
declare module '*.mp3';
```

src/index.ts

```ts
import notificationBellSound from 'assets/sounds/notification_bell.mp3';
const notificationBell = new Audio(notificationBellSound);
notificationBell.play();
```

tsconfig.ts

```json
{
  "compilerOptions": {
    "baseUrl": "src",
    "esModuleInterop": true,
    "isolatedModules": true,
    "lib": ["dom", "dom.iterable", "esnext"],
    "module": "esnext",
    "moduleResolution": "node"
  },
  "include": ["./"]
}

```

Expected: no error.

Actual:

```text
Unlisted dependencies (1)
assets/sounds/notification_bell.mp3  src/index.ts
```

Related issue: https://github.com/webpro/knip/issues/376

**Comment by EvgenyOrekhov** [comment]
Knip version: 5.0.0
Node.js version: 20.10.0

**Comment by EvgenyOrekhov** [comment]
As a workaround, can use relative import: `import notificationBellSound from './assets/sounds/notification_bell.mp3';`

**Comment by EvgenyOrekhov** [comment]
Just tried that, it fixes the issue! Thanks!

Would be great to have that in the docs!

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v5.1.3-canary.0. See [Release 5.1.3-canary.0](https://github.com/webpro/knip/releases/tag/5.1.3-canary.0) for release notes._

**Comment by webpro** [comment]
I've added `mp3` to the list so you don't have to :)
